scripts/bundleConfig.js

```diff
@@ -0,0 +1,27 @@
+const fs = require('fs')
+const config = require('../shared/data/constants.js')
+const fetch = (...args) => import('node-fetch').then(({ default: fetch }) => fetch(...args))
+
+const outputEtagFile = './shared/data/etags.json'
+const bundlePath = './shared/data/bundled/'
+const configProps = config.tdsLists.find((c) => c.name == 'config');
+const etags = {};
+
+async function getConfig(name, url) {
+    let response = await fetch(url)
```

Comment:
My IDE is suggesting all these linting improvements, sorry for being nitty 🤓.

```suggestion
async function getConfig (name, url) {
    const response = await fetch(url)
```